### ISSUE TITLE
TypeError: 'module' object is not callable (from mlxtend.evaluate import bias_variance_decomp)

### ISSUE THREAD

**Issue by Bahar1978** [rasbt/mlxtend#865]
I could always use  from mlxtend.evaluate import bias_variance_decomp) but today when I wanted to use this function , I faced the typeError:
I tried the example of http://rasbt.github.io/mlxtend/user_guide/evaluate/bias_variance_decomp/ and also https://machinelearningmastery.com/calculate-the-bias-variance-trade-off/ that worked before for me but today following both ways I got the error.
Could you please let me know if you updated something regarding to bias_variance_decomp?

**Comment by rasbt** [comment]
Can you print the mlxtend version?

```python
import mlxtend
print(mlxtend.__version__)
```

**Comment by Bahar1978** [comment]
Thanks for the fast reply. Here is the result:  
**Requirement already satisfied: mlxtend in /usr/local/lib/python3.7/dist-packages (0.19.0)**
But now I just used "import mlxtend" and then checked the version and it worked. I just used the following code that did not work:
**from mlxtend.evaluate import bias_variance_decomp**
So it works if I first "import mlxtend".
Thanks again and there is no issue :).
